bucket/warp-terminal.json

```
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://raw.githubusercontent.com/ScoopInstaller/Scoop/master/schema.json",
+    "version": "0.2025.02.24.20.50.stable_04",
+    "description": "Modern, Rust-based terminal with AI built in so you and your team can build great software, faster.",
+    "homepage": "https://www.warp.dev/",
+    "license": {
+        "identifier": "Unknown",
+        "url": "https://github.com/warpdotdev/Warp/discussions/400"
```

```suggestion
        "identifier": "Proprietary",
        "url": "https://www.warp.dev/terms-of-service"
```

It's free but proprietary.